core/src/main/java/org/eclipse/hono/util/Constants.java

```diff
@@ -203,15 +203,22 @@ public Authorities getAuthorities() {
         }
     };
 
+
+    /**
+     * The header name defined for setting the <em>time to deliver</em> for device command readiness notification events.
+     */
+    public static final String HEADER_TIME_TILL_DISCONNECT = "hono-ttd";
+
     /**
      * The header name defined for setting the <em>time to deliver</em> for device command readiness notification events.
+     * 
+     * @deprecated Use {@link #HEADER_TIME_TILL_DISCONNECT} instead.
```

Comment:
FMPOV *TIME_TIL_DISCONNECT* is the correct spelling, since it is the short form of *TIME_UNTIL_DISCONNECT* or is there another reason to change the name of the constant?

Reply:
It looks like there's the 3 choices `until`, `till` and `'til` (see [here](https://www.merriam-webster.com/words-at-play/should-you-use-until-or-till-or-til)). 
Since we can't put the apostrophe in the constant here, I would opt for `till` with 2 L as well.

Reply:
IMHO "til" is definitely wrong:

* https://www.lexico.com/en/definition/till
* https://www.lexico.com/en/definition/until
* https://www.lexico.com/en/definition/til

So I would go for "till" or  "until". I don't mind having typos in various places, but I think in the more official APIs, we should fix it.